coopterrones/rancid-tomatoes
# App unit test

# Pull Request


#### What's this PR do?
1. Adds integration tests in `app.test.js` NOTE: Please excuse the poor naming on this single commit called 'unit' test.. Because we are testing the full app , this is an integration test for all components within app.

#### Where should the reviewer start?
*line 1 in `app.test.js` 

#### How should this be manually tested?
*Check for all passing tests
*Carefully look through each it block to check for adequate testing of a full rendered app 

#### Any background context you want to provide?
*With all unit tests factored in as passing, this integration should cover our rendered app. We should think about what other testing we might want to consider. 

#### Screenshots (if appropriate):


#### Questions
